understanding/use-of-color.html

```diff
@@ -86,16 +109,7 @@ <h2>事例</h2>
                <li><p><strong>試験</strong></p>
                   
                   
-                  <p>学生は、化学化合物の SVG 画像を見て、図に使用されている色や数字に基づいて、存在する化学の要素を識別する。各要素に関連付けられたテキストによる代替は、要素の色に名前を付けて、図で要素の位置を示している。色を知覚できない学生は、クラスメートと同じように、化合物に関する同じ情報を得る。(この達成方法は、<a href="non-text-content.html" class="guideline">ガイドライン 1.1</a> のレベル A も満たす)</p>
-                  
-                  
-               </li>
-               
-               
-               <li><p><strong>使用不可になっているフォーム要素</strong></p>
-                  
-                  
-                  <p>マークアップ又はスクリプトによって使用不可になっているフォーム要素は、ユーザエージェントによってグレーアウトされ、使用できない。使用不可の状態では、これらの要素はフォーカスを受け取らない。支援技術は、使用不可の状態になっている要素をプログラムが解釈できる。色の変化及びフォーカスがあたらないことによって、コントロールの状態に関する情報を視覚的に提供している。</p>
+				<p>学生は、化合物の SVG 画像を見て、図に使用されている色及び各元素に隣接する番号の<strong>両方</strong>に基づいて、存在する化学元素を識別する。凡例によって、元素の種類ごとの色及び番号が示されている。目が見えてもすべての色の違いを知覚できない利用者は、それでも番号を頼りに画像を理解することができる。</p>
```

Comment:
元素の横の番号が示されている、化合物の図というのがピンときませんが…まあそう書いてるので仕方ないですね。

```suggestion
				<p>学生は、化合物の SVG 画像を見て、図に使用されている色及び各元素に隣接する番号の<strong>両方</strong>に基づいて、存在する化学元素を識別する。凡例によって、元素の種類ごとの色及び番号が示されている。目が見えてもすべての色の違いを知覚できない利用者でも、番号を頼りに画像を理解することができる。</p>
```